guide/glossary.md

```diff
@@ -7,14 +7,180 @@ nav_order: 2
 # Glossary
 {: .no_toc }
 
-### Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+## Address
 
+## Account
+
+Just like wallet, the term account can also be used for very different things. In bitcoin wallets that follow the hierarchy described in BIP 44, a bitcoin wallet can have multiple accounts, with each one having its own addresses. However, account is also oftentimes used for accounts with third-party service providers.
+
+Differentiate between
+
+- Bitcoin wallet account
+- Service account
+
+## Bech32
+
+## Bitcoin / bitcoin
+
+## Bitcoin Core
+
+## Bitcoin Core GUI
+
```

Comment:
The graphical user interface (GUI) for the Bitcoin Core software which includes a full node and wallet functionality.